examples/key-export/src/pages/index.tsx

```diff
@@ -0,0 +1,213 @@
+                  <p>{val.privateKeyName}</p>
+                </td>
+                <td className={styles.cell}>
+                  <p className={styles.idCell}>{val.privateKeyId}</p>
+                </td>
+                <td className={styles.cell}>
+                  <p>{formatCurve(val.curve!)}</p>
+                </td>
+                <td className={styles.cell}>
+                  <p className={styles.addressCell}>
+                    {val.addresses[0].address!}
+                  </p>
+                </td>
+              </tr>
+            );
+          })}
+        </tbody>
+      </table>
+    </div>
+  );
+
+  const exportKey = async (keyId: string) => {
+    if (iframeStamper === null) {
+      throw new Error("cannot export private key without an iframe");
+    }
+
+    const response = await axios.post("/api/exportPrivateKey", {
+      privateKeyId: keyId,
+      targetPublicKey: iframeStamper.publicKey(),
+    });
+
+    let injected = await iframeStamper.injectKeyExportBundle(
+      response.data["exportBundle"]
+    );
+    if (injected !== true) {
+      throw new Error("unexpected error while injecting export bundle");
+    }
+
+    setShowPrivateKey(true);
+  };
+
+  return (
+    <main className={styles.main}>
+      <a
+        href="https://www.turnkey.com"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <Image
+          src="/logo.svg"
+          alt="Turnkey Logo"
+          className={styles.turnkeyLogo}
+          width={100}
+          height={24}
+          priority
+        />
+      </a>
+
+      {!iframeStamper && <p>Loading...</p>}
+      {iframeStamper && iframeStamper.publicKey() && privateKeys.length > 0 && (
+        <div>{PrivateKeysTable}</div>
+      )}
+      {selectedPrivateKey && (
+        <div className={styles.copyKey}>
+          <h2>Private key</h2>
+          <p>
+            You are about to reveal your private key. By revealing this private
+            key you understand that:
+          </p>
+          <ul>
+            <li>
+              <p>Your private key is the only way to recover your funds.</p>
+            </li>
+            <li>
+              <p>Do not let anyone see your private key.</p>
+            </li>
+            <li>
+              <p>
+                Never share your private key with anyone, including Turnkey.
+              </p>
+            </li>
+          </ul>
+          <div className={styles.reveal}>
+            <button
+              className={styles.revealButton}
+              onClick={() => {
+                exportKey(selectedPrivateKey);
+              }}
+            >
+              Reveal
+            </button>
+          </div>
+        </div>
+      )}
+      <div
+        style={{ display: showPrivateKey ? "block" : "none" }}
+        id={TurnkeyIframeContainerId}
+      ></div>
+    </main>
+  );
+}
+function assertNever(curve: string) {
+  throw new Error("Function not implemented.");
```

Comment:
given this is used in the case where we have an unknown curve, the message should be "Unknown curve: ${curve}" or if you want to keep it generic "case not implemented"